cloudability/metrics-agent
# CLDYCON-3974 E2E testing for Kubernetes 1.28

#### What does this PR do?

- Include E2E testing for Kubernetes 1.28
- Disable SonarCloud and remove references

#### Where should the reviewer start?

#### How should this be manually tested?

- `make test-e2e-1.28`, but not really necessary since you can verify the results of the GitHub "Test E2E AMD" build step.

#### Any background context you want to provide?

#### What picture best describes this PR (optional but encouraged)?

#### What are the relevant Github Issues?

- https://jira.apptio.com/browse/CLDYCON-3974

#### Developer Done List


#### For the Reviewer:

#### By approving this PR, the reviewer acknowledges that they have checked all items in this done list.

#### Reviewer/Approval Done List
